pkg/kn/commands/broker/create.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	clientv1beta1 "knative.dev/client/pkg/eventing/v1beta1"
+	"knative.dev/client/pkg/kn/commands"
+)
+
+var create_example = `
+# Create a broker 'mybroker' in the current namespace
+  kn broker create mybroker
+# Create a broker 'mybroker' in the 'myproject' namespace
+  kn broker create mybroker --namespace myproject
+`
+
+func NewBrokerCreateCommand(p *commands.KnParams) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "create NAME",
+		Short:   "Create a broker.",
```

Comment:
```suggestion
		Short:   "Create a broker",
```

Please don't use punctuation in the descriptions.